extensions/vscode/src/views/homeView.ts

```diff
@@ -999,6 +1008,16 @@ export class HomeViewProvider implements WebviewViewProvider, Disposable {
         // Should we not push deployments with no config or matching credentials?
         deployments.push(deployment);
       });
+      deployments.push({
+        label: "New",
+        kind: QuickPickItemKind.Separator,
+        lastMatch: false,
+      });
+      deployments.push({
+        iconPath: new ThemeIcon("plus"),
+        label: createNewDeploymentLabel,
+        lastMatch: includedContentRecords.length ? false : true,
+      });
```

Comment:
I had a list of deployments in my connect-content directory, and with this at the bottom I had to scroll to see it. Maybe it can be at the top of the list?

Reply:
I have implemented this, but I have split it off to a separate issue (https://github.com/posit-dev/publisher/issues/2110) and PR.